plugins/Demo/demoAddEmFunction/assets/demoAddEmFunction.js

```diff
@@ -0,0 +1,19 @@
+function sayHello(message) {
```

Comment:
+1 for adding plugin demo!

Reply:
Maybe adding javascript Class for function :/